simple_salesforce/login.py

```diff
@@ -154,7 +155,10 @@ def SalesforceLogin(
             consumer_key is not None and \
             privatekey_file is not None:
         header = {'alg': 'RS256'}
-        expiration = datetime.utcnow() + timedelta(minutes=3)
+        dt = datetime.utcnow()
+        if set_exp_to_local:
```

Comment:
Could consolidate these three lines to a singular line

`dt = datetime.now(timezone.utc).astimezone() if set_exp_to_local else datetime.utcnow()`

Other than that I am good to merge this